stable/kong/values.yaml

```diff
@@ -251,6 +251,10 @@ enterprise:
 # Set runMigrations to run Kong migrations
 runMigrations: true
 
+# set maxSurge and maxUnavailable
+maxSurge: "25%"
+maxUnavailable: "25%"
```

Comment:
The default replica count is 1. How will these values work affect rollout in such a case?
I tried reading the doc but couldn't figure out:
https://kubernetes.io/docs/concepts/workloads/controllers/deployment/

My concern here:
Would the rollout be blocked because both conditions can never be satisfied with a replica count of 1 (or even 2)?

Reply:
You are right. 
Just chose the kubernetes defaults without thinking.
Would maxSurge: 100%, maxUnavailable: 0% be good default values?
or 100% / 100% ?